content/Arduino Cloud/Cloud Editor/Import-your-sketchbook-and-libraries-to-the-Cloud-Editor.md

```diff
@@ -3,48 +3,46 @@ title: "Import your sketchbook and libraries to the Cloud Editor"
 id: 360011890519
 ---
 
-It is possible to import easily all your sketches and libraries from Arduino IDE (desktop Arduino software) to Arduino Cloud Editor (online version of Arduino IDE) in just a few clicks. This comes very handy as all your projects will be saved in Arduino Cloud and are accessible from any computer.
+You can easily transfer all your sketches and libraries from the Arduino IDE to the Arduino Cloud Editor with just a few clicks. This feature enables you to secures your projects in the Arduino Cloud, giving you access from any computer.
 
-In addition, Arduino Cloud Editor allows you to import sketches and files in the following formats:
+Arduino Cloud Editor allows you to import sketches and files in the following formats:
 
-* Single sketches in `.ino`, `.pde` and `.zip` format.
-* Libraries in `.zip` format.
-* Zipped folders containing sketches and libraries.
+- Single sketches in `.ino`, `.pde`, and `.zip` format.
+- Libraries in `.zip` format.
+- Zipped folders containing sketches and libraries.
 
 > Note: Make sure your libraries are in a folder called ‘libraries’. Be sure not to mix sketches and libraries in the same folder.
 
-![Import files gif](img/ImportFiles.gif)
-
 ## Steps
 
-1. Find your sketchbook:
+1. Find your sketchbook
 
-   On your PC find out where your sketchbook folder is (it is called ‘Arduino’, unless you renamed it) and you will find it under the following directory:
+   On your PC find out where your sketchbook folder is (it is called ‘Arduino’ unless you renamed it) and you will find it under the following directory:
 
-   * Windows: it’s usually under `My Computer/Documents/Arduino`
-   * macOS: `User/Documents/Arduino`
-   * Linux: `$HOME/Arduino`.
+   - Windows: it’s usually under `This PC/Documents/Arduino`
+   - macOS: `User/Documents/Arduino`
+   - Linux: `$HOME/Arduino`.
 
-2. Zip your sketchbook:
+1. Zip your sketchbook
 
-   Make a `.zip` pack of your sketchbook, you should obtain a file called `Arduino.zip`. Make sure it is `.zip` format, any other archive formats will not work.
+   Make a `.zip` pack of your sketchbook, you should obtain a file called `Arduino.zip`. Make sure it is in `.zip` format, any other archive formats will not work.
 
-   ![Create ZIP file](img/ImportFiles1.jpg)
+   ![Create ZIP file](img/ImportFiles1.jpg)
```

Comment:
On another note, I would just remove the image:

* It's inconsistent with other images in the article (bad compression, different drop-shadow)
* It's instructive only for macOS users